src/System.Private.Xml/tests/XmlReader/XmlResolver/XmlSystemPathResolverTests.cs

```diff
@@ -92,7 +92,7 @@ public static void TestResolveInvalidPaths()
         [Fact]
         public static void TestResolveInvalidPath()
         {
-            Assert.Throws<System.Net.WebException>(() => XmlReader.Create("ftp://www.bing.com"));
+            Assert.Throws<System.Net.WebException>(() => XmlReader.Create("ftp://host.invalid"));
```

Comment:
Seems like this is now technically testing something slightly different, as was discussed in https://github.com/dotnet/corefx/pull/19818, but maybe the difference here doesn't matter?  The perf improvement in the test is certainly welcome.